dbos_transact/context.py

```diff
@@ -3,15 +3,42 @@
 import os
 import uuid
 from contextvars import ContextVar
+from enum import Enum
 from types import TracebackType
-from typing import TYPE_CHECKING, Literal, Optional, Type
+from typing import TYPE_CHECKING, Literal, Optional, Type, TypedDict
+
+from opentelemetry.trace import Span, Status, StatusCode
 
 if TYPE_CHECKING:
     from .fastapi import Request
 
 from sqlalchemy.orm import Session
 
 from .logger import dbos_logger
+from .tracer import dbos_tracer
+
+
+# Values must be the same as in TypeScript Transact
+class OperationType(Enum):
+    HANDLER = "handler"
+    WORKFLOW = "workflow"
+    TRANSACTION = "transaction"
+    COMMUNICATOR = "communicator"
+    PROCEDURE = "procedure"
+
+
+# Keys must be the same as in TypeScript Transact
+class TracedAttributes(TypedDict, total=False):
+    name: str
+    operationUUID: Optional[str]
+    operationType: Optional[str]
```

Comment:
You can type this string as a Literal. Example:
```python
OperationTypes = Literal[
    "handler", "workflow", "transaction", "communicator", "procedure"
]
```
This way, mypy will check you only assign the allowed strings to it.